packages/babel-helper-fixtures/src/index.ts

```diff
@@ -119,7 +122,7 @@ function pushTask(taskName, taskDir, suite, suiteName) {
   const taskOptsLoc = tryResolve(taskDir + "/options");
   if (taskOptsLoc) Object.assign(taskOpts, require(taskOptsLoc));
 
-  const test = {
+  const test: any = {
```

Comment:
Doesn't TS infer the correct type here?

Reply:
it does, but there are additional properties assigned later(`sourceMappings`, `sourceMap`, `inputSourceMap`)

other options would be - restructure the code to construct `test` with all properties in it, or to have type for it separately

Reply:
made a bit better fix instead of any